codegen/gql_code_builder/lib/data.dart

```diff
@@ -19,6 +20,9 @@ Library buildDataLibrary(
     generateMaybeWhenExtensionMethod: false,
   ),
 ]) {
+  final fragmentMap = _fragmentMap(docSource);
+  final dataClassAliasMap = _dataClassAliasMap(docSource, fragmentMap);
```

Comment:
Okay, to make this configurable, we can update this line. bew gql_build option can be added for backward compatibility.
e.g
```dart
final dataClassAliasMap = featureEnable ? _dataClassAliasMap(docSource, fragmentMap) : <String, Reference>{};
```